packages/Adapters/POD/src/CMakeLists.txt

```diff
@@ -23,7 +21,22 @@ APPEND_SET(SOURCES
   ) 
 
 TRIBITS_ADD_LIBRARY(
-  dtk_c_api
+  dtk_pod
   HEADERS ${HEADERS}
   SOURCES ${SOURCES}
   )
+
+TRIBITS_ADD_LIBRARY(
```

Comment:
Why do we have to split up the C/C++ code? Isn't the extern "C" good enough for linking with other C codes?